wgpu-core/src/device/mod.rs

```diff
@@ -1189,7 +1189,9 @@ impl<A: HalApi> Device<A> {
                 break 'b Err(TextureViewNotRenderableReason::Usage(texture.desc.usage));
             }
 
-            if resolved_dimension != TextureViewDimension::D2 {
+
+            if resolved_dimension != TextureViewDimension::D2
+                && !(self.features.contains(wgt::Features::MULTIVIEW)){
```

Comment:
I think this should still check that the view dimension is `D2Array` if the feature is available instead of allowing all through.